### ISSUE TITLE
[Request] DeclareStrictTypes Add Ability to Specify NewLines After

### ISSUE THREAD

**Issue by virgofx** [slevomat/coding-standard#208]
I noticed a few cases where the `namespace` declaration was immediately adjacent to the `declare` statement in our code bases and couldn't find any existing sniffs to handle this. Would it be possible to add it as a new parameter option for `DeclareStrictTypes` ?

For example: If the parameter was labeled `newLinesAfterDeclareStatement` and set to 1, I would like to enforce:

```php
<?php
declare(strict_types = 1);
namespace Foo\Bar;
```
to
```php
<?php
declare(strict_types = 1);

namespace Foo\Bar;
```

**Comment by kukulich** [comment]
See https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/PSR2/Sniffs/Namespaces/NamespaceDeclarationSniff.php

**Comment by virgofx** [comment]
That allows you to ensure a blank line **after** the namespace, but not before?

**Comment by Majkl578** [comment]
I think this should rather be namespace-related sniff (older code can't have `declare(strict_types=1)` due BC), but agreed, it's needed. The one referenced above is insufficient.

**Comment by kukulich** [comment]
I hope I'll add the new option this week.

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/52cca43f4ef5a4fd69b416174f82b363e1915b7d

**Comment by virgofx** [comment]
:+1:

**Comment by Majkl578** [comment]
👍, although I'd still prefer this separately for T_NAMESPACE, to be useful for legacy untyped code as well. :) Any ETA for 4.2 @kukulich?

**Comment by kukulich** [comment]
@Majkl578 Today or tomorrow :) I'm trying to solve as much issues as possible :)

**Comment by Majkl578** [comment]
Awesome! Nice work, I will try to send one or two more PRs today, but don't wait for me. :)

**Comment by kukulich** [comment]
@Majkl578 I'll wait :) I need to put aside this repo for a while after the release so I can work on something else :)

I'll be happy if you can test master if there are no new bugs.

**Comment by Majkl578** [comment]
> I'll be happy if you can test master if there are no new bugs.

Nothing new found in my feature set compared to 4.0.0. 👍
